sphinx/README.txt

```diff
@@ -11,6 +11,16 @@ To build the gallery, ggplot and seaborn are also needed:
     $ pip install ggplot
     $ pip install seaborn
 
+
+Manual actions for new releases
+===============================
+
+In ``conf.py`` the links that appear in the docs nav appear. On a new release, the 
+releases link should be changed to the latest release notes (we should find a way to automate this)
```

Comment:
There's a script that updates the versions to the new version for release, I'd suggest it could also note the previous version and insert it in `conf.py` where appropriate.